modules/lazyload/module.php

```diff
@@ -96,12 +114,23 @@ public function __construct() {
 		}
 
 		add_action( 'elementor/element/after_add_attributes', function( Element_Base $element ) {
+			if ( ! $this->is_document_support_lazyload() ) {
+				return;
+			}
 			$this->update_element_attributes( $element );
 		} );
 
-		add_filter('elementor/files/css/selectors', function( $control, $value ) {
+		add_filter('elementor/files/css/selectors', function( $control, $value, $post_id ) {
+			if ( ! $post_id ) {
```

Comment:
```suggestion
			if ( ! isset( $context['post_id'] ) ) {
```

Reply:
I used `$this` and moved the logic to the lazyload module